tests/integration/components/global-navigation/search-test.js

```diff
@@ -79,27 +79,26 @@ module('Integration | Component | global-navigation/search', function (hooks) {
 		await click('.wds-global-navigation__search-submit');
 	});
 
-	// TODO: this test fails, however functionality works
-	// test('search input results are shown after click on Enter', async function (assert) {
-	// 	assert.expect(1);
-
-	// 	this.set('goToSearchResults', function (query) {
-	// 		assert.equal(query, 'query', 'goToSearchResult should be called with query on Enter');
-	// 	});
-
-	// 	await render(hbs`
-	// 	{{global-navigation/search
-	// 		model=model.search
-	// 		goToSearchResults=goToSearchResults
-	// 		onSearchToggleClicked=onSearchToggleClicked
-	// 	}}`);
-
-	// 	const searchInputSelector = '.wds-global-navigation__search-input';
-
-	// 	await click('.wds-global-navigation__search-toggle-icon');
-	// 	await fillIn(searchInputSelector, 'query');
-	// 	await triggerKeyEvent('.wds-global-navigation__search-container', 'keydown', 13);
-	// });
+	test('search input results are shown after click on Enter', async function (assert) {
```

Comment:
why it started to work?

Reply:
it works with fix in this PR. uncommenting it without fix leads to it not working.